api/python/cellxgene_census/src/cellxgene_census/_open.py

```diff
@@ -4,7 +4,7 @@
 
 """Open census and related datasets
 
-Contains methods to open  publicly hosted versions of Cell Census object and access its source datasets.
+Contains methods to open publicly hosted versions of Cellxgene Census object and access its source datasets.
```

Comment:
```suggestion
Contains methods to open publicly hosted versions of CELLxGENE Census object and access its source datasets.
```

Reply:
@atolopko-czi - the motivating GH issue specifically requests the other capitalization. See slack, where I asked if this is actually correct.